registration-processor/registration-processor-rest-client/src/main/java/io/mosip/registration/processor/rest/client/utils/RestApiClient.java

```diff
@@ -62,6 +58,12 @@
  */
 @Component
 public class RestApiClient {
+	
+	@Value("${registration.processor.httpclient.connections.max.per.host:20}")
+	private int maxConnectionPerRoute;
+
+	@Value("${registration.processor.httpclient.connections.max:100}")
+	private int totalMaxConnection;
 
```

Comment:
Lets change for every pararent also to P1 whenever any one of the child modules have change. also lets ensure to build only the changed modules, other needs to be commented in the pom file to avoid building.

Reply:
done